zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -119,6 +128,14 @@ public static boolean executePublish(Collection collection, CollectionReader col
             success &= publishDatasets(collection);
         }
 
+        if (imageFuture != null) {
+            try {
+                success = success && imageFuture.get();
+            } catch (InterruptedException | ExecutionException e) {
+                success = false;
```

Comment:
I can't remember what was discussed and agreed on what happens to the publish if this fails . I think we said the publish should carry on regardless.

I think we should be logging out these errors for visibility on what went wrong.

Reply:
I have moved the catch block from the image future here so the logs for all exceptions are now handled consistently.